server/background.py

```diff
@@ -0,0 +1,31 @@
+import threading
+import time
+import schedule
+
+
+# Copied from https://schedule.readthedocs.io/en/stable/background-execution.html
+
+
+def run_continuously(interval=1):
```

Comment:
might be nice to name this `interval_s` or `Interval_secs`, etc. so it's clear at a glance what the unit is.